client/app/bundles/course/survey/components/QuestionCard.jsx

```diff
@@ -42,31 +69,61 @@ const styles = {
 };
 
 class QuestionCard extends React.Component {
-  static renderOptions(question, Widget) {
+  static renderOptionsList(question, Widget) {
     const { byWeight } = sorts;
     return (
       <div>
         { question.options.sort(byWeight).map(option => (
           <div key={option.id} style={styles.option}>
             <Widget disabled style={styles.optionWidget} />
-            <div style={styles.optionBody}>
+            <div>
               { option.image_url ?
                 <Thumbnail src={option.image_url} style={styles.image} /> : [] }
-              { option.option ? option.option : '' }
+              { option.option || null }
             </div>
           </div>
         ))}
       </div>
     );
   }
 
+
+  static renderOptionsGrid(question, Widget) {
+    const { byWeight } = sorts;
+    return (
+      <div style={styles.grid}>
+        { question.options.sort(byWeight).map(option => (
+          <Card
```

Comment:
Might be good to make this an "inline" sub-component to avoid unnecessary re-rendering?   Otherwise I think all options will be re-rendered if one of them changes.